File: web/Models/SearchFlight.cs
namespace BookingFlyWebApp.Models;

public class SearchFlight
{
    public string startDate { set; get; }
    public string endDate { set; get; }
    public string departure { set; get; }
    public string arrival { set; get; }
    public int adultQuantity{ set; get; }
    public int childQuantity{ set; get; }
}

File: java/ticket_Service/src/main/java/org/tasc/booking/ticket_Service/dto/Requet/SearchFlight.java
package org.tasc.booking.ticket_Service.dto.Requet;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SearchFlight {
    public String startDate;
    public String endDate;
    public String departure;
    public String arrival;
    public int adultQuantity = 1;
    public int childQuantity = 0;

}
